specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/examples/dispatch/SuccessfulAddConnectedServicesRequest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "Ocp-Apim-Subscription-Key": "00000000000000000000000000000000",
+    "appId": "86226c53-b7a6-416f-876b-226b2b5ab07b",
+    "versionId": "0.1",
+    "connectedServices": {
+      "type": "luis",
+      "intentName": "intent1",
+      "name": "child app",
+      "version": "0.1",
+      "region": "westus"
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+      },
+      "body": {        
```

Comment:
Is it normal that the response body is empty?

Reply:
This is the case